content/blog/posts/osl-alumni-matthew-tesla.rst

```diff
@@ -0,0 +1,18 @@
+OSL Alumni Matthew Johnson on working at Tesla
+==============================================
+:date: 2019-03-07
+:slug: osl-alumni-matthew-johnson
+:author: Jerry Peng
+:img: matthew.jpg
+:tag: featured-stories
+
+Matthew Johnson is an OSL alumni who currently works at Tesla as a senior software engineer.
+He recently sat down with College of Engineering to talk about his experience working at Tesla.
+
+`YouTube Video: Graduate Matthew Johnson`_
```

Comment:
I'm wondering how difficult it will be to convert this to an embedded video so you could just play the video in the html page.